packages/adapter-static/index.js

```diff
@@ -7,7 +7,7 @@ export default function (options) {
 		name: '@sveltejs/adapter-static',
 
 		async adapt(builder) {
-			if (!options?.fallback) {
+			if (!options?.fallback && builder.config.kit.router.type !== 'hash') {
```

Comment:
This breaks everyone not on the latest version of kit, needs to use optional chaining for the type part

Reply:
ah shit, this is what i get for trying to get stuff launched while distracted on a train with only a dwindling cellphone hotspot for connectivity. will open a PR